### PR TITLE
chore: Fix async is null error in test runner

### PR DESCRIPTION
**Pull request by WilcoFiers**
This one has been randomly crashing our test runner, depending on if the execution of the checks test took more than 500ms or not.

### CODE REVIEW

**Comment by marcysutton** [comment]
@WilcoFiers how can we test to make sure this works?

**Comment by WilcoFiers** [comment]
@marcysutton Tests are passing. What do you mean?

**Comment by marcysutton** [comment]
I mean how can we verify that the issue is resolved, since it only happened occasionally? Just watch Circle?

File: test/checks/media/frame-tested.js [hunk 1]
```diff
@@ -16,49 +13,50 @@ describe('frame-tested', function () {
...
 			[iframe.contentWindow, 'axe.ping', null, undefined]);
 		assert.isFunction(respondableCalls[0][4]);
 	});
 
-	it('passes if the iframe contains axe-core', function (done) {
-		checkEvaluate(iframe, { timeout: 20 });
+	it('passes if the iframe contains axe-core', function () {
 		checkContext._onAsync = function (result) {
 			assert.isTrue(result);
-			done();
```
[the remark this answers is not in the document]
Comment (WilcoFiers) [reply test/checks/media/frame-tested.js]: I've updated these :)

**Pull request merged**